src/rcheevos/rc_client.c

```diff
@@ -1433,6 +1433,31 @@ static void rc_client_copy_achievements(rc_client_load_state_t* load_state,
   subset->achievements = achievements;
 }
 
+static uint8_t rc_client_map_leaderboard_format(const rc_api_leaderboard_definition_t* defn)
+{
+  switch (defn->format) {
+    case RC_FORMAT_SECONDS:
+    case RC_FORMAT_CENTISECS:
+    case RC_FORMAT_MINUTES:
+    case RC_FORMAT_SECONDS_AS_MINUTES:
+      return RC_CLIENT_LEADERBOARD_FORMAT_TIME;
+
+    case RC_FORMAT_SCORE:
+      return RC_CLIENT_LEADERBOARD_FORMAT_SCORE;
+
+    case RC_FORMAT_FRAMES:
```

Comment:
FRAMES should be a TIME format. It's implicitly divide by 60 and format as CENTISECS.